pkg/entity/datapolicy/texts.go

```diff
@@ -59,6 +59,46 @@ data_policy:
 ...
 `)
 
+var evaluateLongDocs = LongDocs(`
+Evaluates an existing data policy by applying it to sample data provided in a csv file.
+You can use this to verify the correctness of your field transforms and filters.
+The csv file should contain a header row with the column names, matching the fields in the data policy.
+A comma should be used as the delimiter.
```

Comment:
This leads me to think that the `CSV` sample data in the proto should be more configurable. Maybe not now, but later. Therefore, the csv should not be a `string` in the proto field, but a message.